packages/video_player/video_player/example/integration_test/video_player_test.dart

```diff
@@ -162,5 +165,23 @@ void main() {
       await tester.pumpAndSettle();
       expect(_controller.value.isPlaying, true);
     }, skip: kIsWeb); // Web does not support local assets.
+
+    testWidgets('test video player using static file() method as constructor',
+        (WidgetTester tester) async {
+      String tempDir = (await getTemporaryDirectory()).path;
+      ByteData bytes = await rootBundle.load('assets/Butterfly-209.mp4');
+
+      File file = File('$tempDir/Butterfly-209.mp4');
+      await file.writeAsBytes(bytes.buffer.asInt8List());
+
+      VideoPlayerController fileController = VideoPlayerController.file(file);
```

Comment:
there is a note on the homepage that this will throw unimplementederror on web, does it need a platform check?

Reply:
This test is ignored on web as I don't believe it will work. See line 185 for ignored web platform 😄

Reply:
That makes sense then - and obviously a failed test in CI is a sort of "fail fast" (so not expensive to fix) issue in case it is enabled later. No objections then